### PR TITLE
update disk issues

### PR DESCRIPTION
**Pull request by andyzhangx**
/assign @feiskyer @karataliu 

cc @weherdh I put the azure disk & azure file issues here

### CODE REVIEW

File: docs/azuredisk-issues.md [hunk 1]
```diff
@@ -236,6 +239,7 @@ spec:
     runAsUser: 0
     fsGroup: 0
 ```
+ > Note: Since gid & uid is mounted as 0(root) by default, if set as non-root(e.g. 1000), k8s will use chown/chmod to change all dir/files under that disk, this is a time consuming job, whch would make mount device very slow, in this issue: [Timeout expired waiting for volumes to attach](https://github.com/kubernetes/kubernetes/issues/67014#issuecomment-413546283), it costs about 10 min for chown/chmod operation complete.
```
Comment (karataliu) [inline docs/azuredisk-issues.md]: whch -> which

File: docs/azuredisk-issues.md [hunk 2]
```diff
@@ -319,3 +323,44 @@ This is a common k8s issue, other cloud provider would also has this issue. Ther
 
 **Work around**:
 delete pod first and then delete azure disk pvc after a few minutes
+
+### 12. create azure disk PVC failed due to account creation failure
+ > pls note this issue only happens on **unmanaged** k8s cluster
```
Comment (karataliu) [inline docs/azuredisk-issues.md]: pls -> please

File: docs/azuredisk-issues.md [hunk 3]
```diff
@@ -319,3 +323,44 @@ This is a common k8s issue, other cloud provider would also has this issue. Ther
...
+| ---- | ---- |
+| v1.9 | no such issue |
+| v1.10 | no such issue |
+| v1.11 | 1.11.3 |
+| v1.12 | no such issue |
+
+**Work around**:
+ - create a storage account and specify that account in azure disk storage class, e.g.
+ ```
+ kind: StorageClass
```
Comment (karataliu) [inline docs/azuredisk-issues.md]: additional space before kind

File: docs/azuredisk-issues.md (hunk 2 above)
Comment (feiskyer) [reply docs/azuredisk-issues.md]: seems we have met such typos for many times, it's better to add a spell checking in our CI

**Comment by feiskyer** [comment]
A reference of verify-spelling is https://github.com/kubernetes/kubernetes/blob/master/hack/verify-spelling.sh

File: docs/azuredisk-issues.md (hunk 1 above)
Comment (karataliu) [reply docs/azuredisk-issues.md]: only chown? when does chmod applies?

Comment (andyzhangx) [reply docs/azuredisk-issues.md]: depends on what value gid or uid is set, k8s will use chown/chmod to apply.

File: docs/azuredisk-issues.md (hunk 2 above)
Comment (andyzhangx) [reply docs/azuredisk-issues.md]: fixed

File: docs/azuredisk-issues.md (hunk 1 above)
Comment (andyzhangx) [reply docs/azuredisk-issues.md]: fixed

File: docs/azuredisk-issues.md (hunk 3 above)
Comment (andyzhangx) [reply docs/azuredisk-issues.md]: fixed

**Comment by weherdh** [comment]
@andyzhangx Thanks Andy to keep updating me! I really need the notification.

**Pull request merged**